tensorrt_llm/_torch/attention_backend/trtllm.py

```
@@ -1752,6 +1760,455 @@ def update_quant_config(self, new_quant_config: Optional[QuantConfig]):
+        tokens_per_block = metadata.tokens_per_block
+
+        offset_q = 0
+        offset_kv = 0
+        q_batch_indices = []
+        query_positions = []
+        seq_lens = []
+        per_sample_state = []
+        dense_value_states_per_seq = []
+        dense_value_start_positions = []
+        for sample_idx, (seq_len, seq_len_kv) in enumerate(
+                zip(metadata.seq_lens, metadata.seq_lens_kv)):
```

Ruff (0.15.13)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)